pom.xml

```diff
@@ -998,6 +998,21 @@ under the License.
 			</build>
 		</profile>
 
+		<profile>
+			<id>github-actions</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<excludedGroups>org.apache.flink.testutils.junit.FailsInGHAContainerWithRootUser</excludedGroups>
```

Comment:
I'll double check this as well because I may have tried that in the past 🤔 

There's also more esoteric options but let's not get into those just yet...